test/unit/builtins/map.spec.ts

```diff
@@ -1,95 +1,79 @@
 import * as util from "../../util";
 
 test("map constructor", () => {
-    const result = util.transpileAndExecute("let mymap = new Map(); return mymap.size;");
-
-    expect(result).toBe(0);
+    util.testFunction("let mymap = new Map(); return mymap.size;").expectToMatchJsResult();
 });
 
 test("map iterable constructor", () => {
-    const result = util.transpileAndExecute(
-        `let mymap = new Map([["a", "c"],["b", "d"]]);
-         return mymap.has("a") && mymap.has("b");`
-    );
-
-    expect(result).toBe(true);
+    util.testFunction`
+        let mymap = new Map([["a", "c"],["b", "d"]]);
+         return mymap.has("a") && mymap.has("b");
+    `.expectToMatchJsResult();
 });
 
 test("map iterable constructor map", () => {
-    const result = util.transpileAndExecute(`
+    util.testFunction`
         let mymap = new Map(new Map([["a", "c"],["b", "d"]]));
         return mymap.has("a") && mymap.has("b");
-    `);
-
-    expect(result).toBe(true);
+    `.expectToMatchJsResult();
 });
 
 test("map clear", () => {
     const mapTS = 'let mymap = new Map([["a", "c"],["b", "d"]]); mymap.clear();';
-    const size = util.transpileAndExecute(mapTS + "return mymap.size;");
-    expect(size).toBe(0);
+    util.testExpression("mymap.size;").setTsHeader(mapTS).expectToMatchJsResult();
```

Comment:
expressions should not end with ; (that's for statements) I think you might be getting lucky this works here